apollo-client/src/main/java/com/ctrip/framework/apollo/spring/config/CompositeConfigPropertySource.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2021 Apollo Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.ctrip.framework.apollo.spring.config;
+
+import com.ctrip.framework.apollo.ConfigChangeListener;
+import com.ctrip.framework.apollo.model.ConfigChangeEvent;
+import org.springframework.core.env.CompositePropertySource;
+import org.springframework.core.env.PropertySource;
+
+/**
+ * @author Shawyeok (shawyeok@outlook.com)
+ */
+public class CompositeConfigPropertySource extends CompositePropertySource {
```

Comment:
maybe itself could be a config change listener?
```suggestion
public class CompositeConfigPropertySource extends CompositePropertySource implements ConfigChangeListener {
```